backend/development.ini.sample

```diff
@@ -209,11 +221,11 @@ api.key = %(basic_setup.api_key)s
 ## LDAP server address
 ; ldap_url = ldap://localhost:389
 ## Bind dn to identify the search
-; ldap_bind_dn = cn=admin,dc=directory,dc=fsf,dc=org
+# ldap_bind_dn = cn=admin,dc=directory,dc=fsf,dc=org
 ## The bind password
-; ldap_bind_pass = toor
+# ldap_bind_pass = toor
 ## Base dn to make queries of users
-; ldap_user_base_dn = ou=Users,dc=directory,dc=fsf,dc=org
+# ldap_user_base_dn = ou=Users,dc=directory,dc=fsf,dc=org
```

Comment:
I think here we need to add line for all parameters with empty value like this:
- `; ldap_bind_dn = `
- `; ldap_bind_pass = `
- `; ldap_user_base_dn = `

What do you think @inkhey ?

Reply:
fixed